src/main/java/com/metamx/metrics/SysMonitor.java

```diff
@@ -449,4 +471,100 @@ public void emit(ServiceEmitter emitter)
       }
     }
   }
+
+  private class SysStats implements Stats
+  {
+    @Override
+    public void emit(ServiceEmitter emitter)
+    {
+      Uptime uptime = null;
+      try {
+        uptime = sigar.getUptime();
+      }
+      catch (SigarException e) {
+        log.error(e, "Failed to get Uptime");
+      }
+      if (uptime != null) {
+        final Map<String, Long> stats = ImmutableMap.of(
+            "sys/uptime", Double.valueOf(uptime.getUptime()).longValue()
```

Comment:
Do you mind adding getLoadAverage? I think all three l/a of 1/5/15 min make sense.